api/api/settings.py

```diff
@@ -48,7 +48,7 @@
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
-    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware'
```

Comment:
Restore the `,` at the end of the line. This is the preferred style of writing in Django/Python so that we can just append the elements at the end of the list without worrying if we have to put a `,` at the end of the last element.

Ref: https://stackoverflow.com/a/11597911/9890886